sdk/rust/oak/tests/handle_extract_inject.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package tests;
+
+import "handle.proto";
+
+message TestMessage {
+  string other_arbitrary_field = 1;
+  oak.handle.Sender test_sender = 2 [(oak.handle.message_type) = ".tests.TestMessageType"];
```

Comment:
Could we get a new section in [Programming Oak](https://github.com/project-oak/oak/blob/main/docs/programming-oak.md) to describe how to set up and use these protobuf types/extensions at some point please?

Reply:
Yes absolutely! I'll add those after this PR is merged 😄